config/config.yml.example

```diff
@@ -59,7 +59,7 @@ movement:
 plugins:
     # Do not automatically load the specified plugins
     exclude:
-        - socket
+        - pluginname
```

Comment:
The thing is the syntax for nothing ([]) is so much different that I think we need a real example with a fake name. Discussion on slack seems to prove that.